style.css

```diff
@@ -0,0 +1,71 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  max-width: 480px;
+  overflow: hidden;
```

Comment:
[OPTIONAL]

- [ ] It would be nice to remove `overflow: hidden` from the page `body`. This is because this CSS declaration clips the page overflow and renders the rest of the content invisible. 😀

Reply:
Thank you, @DammyShittu!
I'll do that.